portfolio/src/main/java/com/google/sps/authentication/userservice/UserServiceWrapper.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.authentication.userservice;
+
+import com.google.appengine.api.users.UserService;
+
+/** Wrapper that handles the interaction with User Service API. */
+public final class UserServiceWrapper {
+    private final UserService userService;
+
+    public UserServiceWrapper(UserService userService) {
+        this.userService = userService;
+    }
+
```

Comment:
Nit: you might want to add javadoc for these public methods.

Reply:
Fixed